src/integrations/watchers/indexable-author-watcher.php

```diff
@@ -54,7 +54,7 @@ public function __construct( Indexable_Repository $repository, Indexable_Builder
 	public function register_hooks() {
 		\add_action( 'user_register', [ $this, 'build_indexable' ], \PHP_INT_MAX );
 		\add_action( 'profile_update', [ $this, 'build_indexable' ], \PHP_INT_MAX );
-		\add_action( 'deleted_user', [ $this, 'delete_indexable' ] );
+		\add_action( 'deleted_user', [ $this, 'handle_user_delete' ], 10, 2 );
```

Comment:
I am not totally sure how deleted_user works, but would instances of this fail if they only supply 1 argument or will it then just be null?

Reply:
Good point, I added a default value of `null` to the second parameter of `handle_user_delete`!